src/pages/workspace/accounting/xero/XeroTrackingCategoryConfigurationPage.tsx

```diff
@@ -0,0 +1,103 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import ConnectionLayout from '@components/ConnectionLayout';
+import type {MenuItemProps} from '@components/MenuItem';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections';
+import {getTrackingCategory} from '@libs/actions/connections/ConnectToXero';
+import Navigation from '@libs/Navigation/Navigation';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ROUTES from '@src/ROUTES';
+
+function XeroTrackingCategoryConfigurationPage({policy}: WithPolicyProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const policyID = policy?.id ?? '';
+    const {importTrackingCategories, pendingFields} = policy?.connections?.xero?.config ?? {};
+    const {trackingCategories} = policy?.connections?.xero?.data ?? {};
+
+    const menuItems: MenuItemProps[] = useMemo(() => {
+        const availableCategories = [];
+
+        const costCenterCategoryValue = getTrackingCategory(policy, CONST.XERO_CONFIG.TRACKING_CATEGORY_FIELDS.COST_CENTERS)?.value ?? '';
+        const regionCategoryValue = getTrackingCategory(policy, CONST.XERO_CONFIG.TRACKING_CATEGORY_FIELDS.REGION)?.value ?? '';
+        if (costCenterCategoryValue) {
+            availableCategories.push({
+                description: translate('workspace.xero.mapXeroCostCentersTo'),
+                onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_XERO_TRACKING_CATEGORIES_MAP_COST_CENTERS.getRoute(policyID)),
+                title: translate(`workspace.xero.trackingCategoriesOptions.${costCenterCategoryValue.toLowerCase()}` as TranslationPaths),
+            });
+        }
+
+        if (trackingCategories?.find((category) => category.name.toLowerCase() === CONST.XERO_CONFIG.TRACKING_CATEGORY_FIELDS.REGION)) {
```

Comment:
Why do you have different logic for `Cost centers` and `Region`? 

With `if (costCenterCategoryValue) {` and `trackingCategories?.find((category)` ?

Reply:
Forgot to refactor this one line. Updated.